.github/workflows/ci.yaml

```diff
@@ -32,34 +32,27 @@ jobs:
     - name: Check out repo
       uses: actions/checkout@v2
 
-    - uses: balchua/microk8s-actions@v0.2.2
+    - name: Setup operator environment
+      uses: charmed-kubernetes/actions-operator@main
```

Comment:
We'll want to include a line like this after this step:

```yaml
# https://github.com/charmed-kubernetes/actions-operator/issues/27
- run: sudo snap refresh charmcraft --channel latest/candidate
```

That will get us the latest version of `charmcraft`, and will also require the creation of a `charmcraft.yaml` file

Reply:
Done @knkski, see "Install dependencies" step. I also added the `charmcraft.yaml` file.